agr_literature_service/lit_processing/data_ingest/pubmed_ingest/pubmed_search_new_references.py

```diff
@@ -227,10 +225,19 @@ def query_mods(input_mod, reldate):  # noqa: C901
     not_loaded_pmids4mod = {}
     pmids4mod = {}
     pmids4mod['all'] = set()
+
+    pmids_to_exclude_file = search_path + "pmids_to_excude.txt"
+    exclude_pmids = set()
+    with open(pmids_to_exclude_file, "r") as infile_fh:
+        for line in infile_fh:
```

Comment:
same as above, maybe create a function to factorize code?

Reply:
for later